### PR TITLE
Aegisub: 3.2.1 -> 3.2.2

### PR DESCRIPTION
**Pull request by AndersonTorres**
Adding config.aegisub.\* options

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -483,6 +483,12 @@ let
 
   aegisub = callPackage ../applications/video/aegisub {
     wxGTK = wxGTK30;
+    spellcheckSupport = config.aegisub.spellcheck or true;
+    automationSupport = config.aegisub.automation or true;
+    openalSupport     = config.aegisub.openal or false;
+    alsaSupport       = config.aegisub.alsa or true;
+    pulseaudioSupport = config.aegisub.pulseaudio or true;
+    portaudioSupport  = config.aegisub.portaudio or false;
```
Comment (domenkozar) [inline pkgs/top-level/all-packages.nix]: Why name global options differently than the package ones?

**Comment by domenkozar** [comment]
@AndersonTorres travis shows that the build fails.

**Comment by vbgl** [comment]
I cannot reproduce travis failure : it builds and runs fine on fedora linux x86_64 with chroot.

File: pkgs/top-level/all-packages.nix (hunk 1 above)
Comment (AndersonTorres) [inline pkgs/top-level/all-packages.nix]: No special reason. I prefer without the "Support" suffix on them.

Also, about the failure, I don't know why. Here in my local machine it builds fine.

**Comment by AndersonTorres** [comment]
The Travis error is about "gcc segfault" or something like that.
Also, I renamed the options.

**Comment by pSub** [comment]
Builds fine locally.

**Pull request merged**